constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -273,8 +283,14 @@ export class ClassEditPage extends React.Component {
                             this.handleChange(e, "inputTimeString")
                         }
                     />
-                </div>
 
+                    <h4 className="availability">Class Availability</h4>
+                    <select onChange={(e) => this.onChangeFullState(e)}>
```

Comment:
You can do smoething like this:
```
<select value={this.state.fullState} onChange={...}/>
```
which will match the fullState value at all times. Since fullState is a number (as mentioned before), you may have to do string conversions here because the below options values are all strings of 0, 1, 2.